.github/workflows/pr-check.yml

```diff
@@ -35,3 +35,14 @@ jobs:
       run: |
         cmake -S . -B __build -GNinja -DGREENTEA_CLIENT_STDIO=OFF
         cmake --build __build
+
+    - name: Install greentea-client
+      run: |
+        cmake -S . -B __build -GNinja -DGREENTEA_CLIENT_STDIO=ON
+        cmake --build __build
+        sudo cmake --install __build
+
+    - name: Test downstream application can use find_package
+      run: |
```

Comment:
Is this run guaranteed to come after the install? Do we need to specify a dependency to ensure good ordering?

Reply:
The steps in a job are always executed sequentially. If we had two jobs where job B depended on job A, then we would need to specify a dependency.